cdap-master/pom.xml

```diff
@@ -744,6 +745,29 @@
                 </configuration>
               </execution>
 
+              <!-- Copy metadata consumer extensions -->
+              <execution>
+                <id>copy-metadata-consumer-ext-data-catalog</id>
```

Comment:
Is the extension guaranteed to be present for all deployments ? For eg: OSS. If not , may be a check is required here?

Reply:
Made the change